tasks/gulp/copy-to-destination.js

```diff
@@ -1,87 +1,105 @@
+const { basename, join } = require('path')
+
 const nunjucks = require('nunjucks')
 const gulp = require('gulp')
-const configPaths = require('../../config/paths.js')
 const postcss = require('gulp-postcss')
+const postcssScss = require('postcss-scss')
 const autoprefixer = require('autoprefixer')
-const taskArguments = require('../task-arguments')
-const filter = require('gulp-filter')
-const fs = require('fs')
-const yamlToJson = require('js-yaml')
-const path = require('path')
+const yaml = require('js-yaml')
 const map = require('map-stream')
+const merge = require('merge-stream')
 const rename = require('gulp-rename')
 
-const scssFiles = filter([configPaths.src + '**/*.scss'], { restore: true })
-const yamlFiles = filter([configPaths.components + '**/*.yaml'], { restore: true })
+const configPaths = require('../../config/paths.js')
+const taskArguments = require('../task-arguments')
 
 gulp.task('copy-files', () => {
-  return gulp.src([
-    configPaths.src + '**/*',
-    '!**/.DS_Store',
-    '!**/*.mjs',
-    '!**/*.test.*',
-    '!' + configPaths.src + 'README.md', // Don't override the existing README in /package
-    '!' + configPaths.components + '**/__snapshots__/**',
-    '!' + configPaths.components + '**/__snapshots__/'
-  ])
-    .pipe(scssFiles)
-    .pipe(postcss([
-      autoprefixer
-    ], { syntax: require('postcss-scss') }))
-    .pipe(scssFiles.restore)
-    .pipe(yamlFiles)
-    .pipe(map(function (file, done) {
-      const fixturesFile = generateFixtures(file)
-      done(null, fixturesFile)
-    }))
-    .pipe(rename(path => {
-      path.basename = 'fixtures'
-      path.extname = '.json'
-    }))
-    .pipe(yamlFiles)
-    .pipe(map(function (file, done) {
-      const macroFile = generateMacroOptions(file)
-      done(null, macroFile)
-    }))
-    .pipe(rename(path => {
-      path.basename = 'macro-options'
-      path.extname = '.json'
-    }))
-    .pipe(yamlFiles.restore)
-    .pipe(gulp.dest(taskArguments.destination + '/govuk/'))
+  return merge(
```

Comment:
Changes to this file have been brought over from https://github.com/alphagov/govuk-frontend/pull/2862